faq.html

```diff
@@ -182,18 +182,24 @@ <h3 id="chrome_web_store">
     <h3 id="mobile_support"> <a href="#mobile_support">Mobile Support ?</a></h3>
     <p>Rather than using LibRedirect, use these apps:</p>
     <ul class="clear">
+      <li>
+        URL Manipulation <span>&#8594;</span>
+        <a href="https://github.com/TrianguloY/UrlChecker">URLCheck</a>,
+        <a href="https://github.com/1fexd/LinkSheet">LinkSheet</a>
+      </li>
       <li>
         YouTube <span>&#8594;</span>
-        <a href="https://newpipe.net/">NewPipe</a>,
-        <a href="https://libretube.dev/">LibreTube</a>
+        <a href="https://libretube.dev/">LibreTube</a>,
+        <a href="https://github.com/lamarios/clipious"></a>,
+        <a href="https://newpipe.net/">NewPipe</a>
       </li>
       <li>
         Twitter <span>&#8594;</span>
         <a href="https://fritter.cc/">Fritter</a>
       </li>
       <li>
         Translate <span>&#8594;</span>
-        <a href="https://manerakai.github.io/simplytranslate_mobile/">SimplyTranslate Mobile</a>
```

Comment:
No need to remove SimplyTranslate Mobile. It's mine bruh.

Reply:
Oopsie 🙈, OK 👍🏿